Documentation/ceph-pool-radosnamespace.md

```diff
@@ -0,0 +1,44 @@
+---
+title: RadosNamespace CRD
+weight: 3610
+indent: true
+---
+
+{% include_relative branch.liquid %}
+
+This guide assumes you have created a Rook cluster as explained in the main [Quickstart guide](quickstart.md)
+
+# CephBlockPoolRadosNamespace CRD
+
+Rook allows creation of Ceph BlockPool
+[RadosNamespaces](https://docs.ceph.com/en/latest/man/8/rbd/) through the
+custom resource definitions (CRDs). BlockPool Rados Namespace is an abstraction
+for a block pool. For more information about BlockPool and namespace refer to
```

Comment:
```suggestion
for a Ceph RADOS namespace. For more information about BlockPool and namespace refer to
```

Reply:
It's an abstraction for a block pool. we can have `N` number of rados namespaces in a pool.

Reply:
sounds good :+1: